docs/docs/en/posts/2023-12-24-formatter.rzk.md

```diff
@@ -0,0 +1,243 @@
+---
+authors:
+  - aabounegm
+date: 2023-12-24
+---
+
+# Rzk Formatter
+
+Version 0.7.0 (and later 0.7.3 with bug fixes) of Rzk has recently been released,
+and it brings with it an exciting new feature: the auto formatter!
+It has been under work for over 3 weeks and came with unique design challenges
+that I will attempt to unpack in this post.
+
+<!-- more -->
+
+## Goal
+
+Basically, the target is to write a program that takes in Rzk code as input and
+makes sure it abides by the style guide with minimal modification otherwise.
```

Comment:
```suggestion
makes sure it abides by the [style guide](https://rzk-lang.github.io/sHoTT/STYLEGUIDE/) (developed mainly by [Fredrik Bakke](https://github.com/fredrik-bakke) and other contributors to the [sHoTT project](http://github.com/rzk-lang/sHoTT)) with minimal modification otherwise.
```